src/components/form/EncounterFormPanel.jsx

```diff
@@ -94,7 +95,7 @@ class EncounterFormPanel extends React.PureComponent {
   }
 
   handleCancel() {
-    if (this.getForm().state === FORM_STATES.EDITING) {
+    if (this.getForm().state === FORM_STATES.VIEWING) {
       // if no existing encounter (ie "Enter" mode) redirect to any back link
```

Comment:
@jomadoye  hmm, it looks like from this comment the Cancel button should already being doing the right think.... that is, if there's no encounter, it treat it like a back link.  But obviously this wasn't working in this case... do you know why?